1-exercises/D-do-while/exercise.js

```diff
@@ -7,9 +7,16 @@
 */
 
 function evenNumbersSum(n) {
-    // TODO
+  i = 0; // not to loop each time
+  total = 0; //a varialbe to add from the point
+  do {
+    i++;
+    total += i * 2; // adding the even number to total
```

Comment:
Dear Jud, I was impressed with how you solved this problem using only two variables. Nice job.